### PR TITLE
Assorted typo fixes.

### PR DESCRIPTION
**Pull request by XhmikosR**
There one case I wasn't sure about, so I didn't change it: `binominal`

```
C:\Users\xmr\Desktop\libsass>git grep binominal
src/ast_helpers.hpp:  // Pseudo-element selectors that has binominals
src/parser.cpp:  // Helper to clean binominal string
src/parser.cpp:          std::string parsed(lexed); // always compacting binominals (as dart-sass)
src/parser.hpp:    Expression* parse_binominal();
```

### CODE REVIEW

File: script/tap-driver [hunk 1]
```diff
@@ -370,7 +370,7 @@ function setup_result_obj(line)
   sub("^(not )?ok[ \t]*", "", line)
 
   # If the result has an explicit number, get it and strip it; otherwise,
-  # automatically assing the next progresive number to it.
+  # automatically assing the next progressive number to it.
```
Comment (nschonni) [inline script/tap-driver]: This might need to be fixed somewhere else first, since it's a GPL'd script that is vendor'd in because of Automake.

Comment (XhmikosR) [reply script/tap-driver]: I'll just drop it from this patch :P

**Comment by XhmikosR** [comment]
I dropped that specific change, I didn't notice it's a generated file.

**Comment by XhmikosR** [comment]
How about `binominal`? Isn't that a typo also?

**Comment by nschonni** [comment]
Not sure, but it does look like it

**Comment by XhmikosR** [comment]
I'll leave [these 4 `binominal` instances](https://github.com/sass/libsass/pull/2986#issue-312721643) for someone else to tackle then.

**Comment by XhmikosR** [comment]
Can someone merge this?

**Pull request merged**